x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -497,25 +506,23 @@ class PackagePolicyService {
 
     for (const id of ids) {
       try {
-        const { packagePolicy, installedPkgInfo } = await this.getUpgradePackagePolicyInfo(
-          soClient,
-          id
-        );
+        const { packagePolicy, packageInfo } = await this.getUpgradePackagePolicyInfo(soClient, id);
 
         const updatePackagePolicy = overridePackageInputs(
           {
             ...omit(packagePolicy, 'id'),
-            inputs: packageToPackagePolicyInputs(installedPkgInfo),
+            inputs: packagePolicy.inputs,
```

Comment:
We swapped the order of `original` and `override` arguments in a few places for consistency's sake which caused some bugs on the frontend w/ update values not persisting. Root of the issue was this `basePackagePolicy` argument having the new package version's inputs of the inputs coming from the saved object package policy.